go/database/mpt/live_trie_test.go

```diff
@@ -960,3 +960,34 @@ func BenchmarkValueInsertionInFileTrie(b *testing.B) {
 		})
 	}
 }
+
+func BenchmarkLiveTrie_WithHighNumberOfDirtyNodes_Flush(b *testing.B) {
+	// running this test for all configurations at once takes too much time
+	// hence we use only the latest one
+	config := S5ArchiveConfig
+	// largest nodes have ~512 bytes - we need to create a cache with 18 GB cache,
+	// hence we need ~ 18*2^30/2^9 = 18*2^21 ~ 38 million
+	cacheSize := 38_000_000
+	// number of accounts need to be ~1/3 smaller because we store leafs and branches
+	numAccounts := cacheSize - (cacheSize / 3)
+	dir := b.TempDir()
+	state, err := OpenGoFileState(dir, config, cacheSize)
```

Comment:
A benchmark targeting a state should not be in `live_trie_test.go` but in `state_test.go`